<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="clearMemoryButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MC</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="divisionButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="one">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="six">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="clearButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="five">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="seven">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="plusButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="nine">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="three">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pointButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="zero">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="two">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="equalButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="eight">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="four">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="minusButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="plusminusButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-/+</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="timesButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="squareRootButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>sqrt</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="clearAllButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Clearall</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="setMemoryButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MS</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="addToMemoryButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>M+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="lessToMemoryButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>M-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="readMemoryButton">
     <property name="minimumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>82</width>
       <height>52</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MR</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLineEdit" name="display">
     <property name="minimumSize">
      <size>
       <width>466</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>466</width>
       <height>40</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
